src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1852,6 +1852,69 @@ limitations under the License.
+            </include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
+                <requireAttribute>SERVER_LIST</requireAttribute>
+                <requireAttribute>CLIENT_LIST</requireAttribute>
+                <requireAttribute>PARTS_LIST</requireAttribute>
+            </include>
+            <include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>BINDING</requireAttribute>
+            </include>
+            <include cluster="Scenes" client="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>SCENE_COUNT</requireAttribute>
+                <requireAttribute>CURRENT_SCENE</requireAttribute>
+                <requireAttribute>CURRENT_GROUP</requireAttribute>
+                <requireAttribute>SCENE_VALID</requireAttribute>
+                <requireAttribute>SCENE_NAME_SUPPORT</requireAttribute>
+                <requireCommand>AddScene</requireCommand>
+                <requireCommand>AddSceneResponse</requireCommand>
+                <requireCommand>ViewScene</requireCommand>
+                <requireCommand>ViewSceneResponse</requireCommand>
+                <requireCommand>RemoveScene</requireCommand>
+                <requireCommand>RemoveSceneResponse</requireCommand>
+                <requireCommand>RemoveAllScenes</requireCommand>
+                <requireCommand>RemoveAllScenesResponse</requireCommand>
+                <requireCommand>StoreScene</requireCommand>
+                <requireCommand>StoreSceneResponse</requireCommand>
+                <requireCommand>RecallScene</requireCommand>
+                <requireCommand>GetSceneMembership</requireCommand>
+                <requireCommand>GetSceneMembershipResponse</requireCommand>
+            </include>
+            <include cluster="Groups" client="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>GROUP_NAME_SUPPORT</requireAttribute>
+                <requireCommand>AddGroup</requireCommand>
+                <requireCommand>AddGroupResponse</requireCommand>
+                <requireCommand>ViewGroup</requireCommand>
+                <requireCommand>ViewGroupResponse</requireCommand>
+                <requireCommand>GetGroupMembership</requireCommand>
+                <requireCommand>GetGroupMembershipResponse</requireCommand>
+                <requireCommand>RemoveGroup</requireCommand>
+                <requireCommand>RemoveGroupResponse</requireCommand>
+                <requireCommand>RemoveAllGroups</requireCommand>
+                <requireCommand>AddGroupIfIdentifying</requireCommand>
+            </include>
+            <include cluster="On/Off" client="false" server="true" clientLocked="true" serverLocked="true"></include>
```

Comment:
Why do we not have requireCommand bits for this one?  I suspect in practice we should not have them for any of the bits above except insofar as this device type overrides the default cluster conformance values.  Which it only does for Identify and On/Off.